### DISCUSSION TITLE
Get the damage that is done by each molotov

### DISCUSSION THREAD

**Discussion by goodmoney-zz** [saul/demofile#254] [Questions] [answered]
I know I can get the molotovs with the event "inferno startburn", but is there a way to figure out how many damage is done by each molotov?

**Comment by master117** [comment]
Why not just add the values from all damage events by that molotov?

**Comment by goodmoney-zz** [comment]
I can use the "player_hurt" event to get the damage, but how do I find the right Molotov? 
There is no entityid.

**Comment by master117** [comment]
I don't think fire has an id / it's not linked to a Molotov.

**Comment by goodmoney-zz** [comment]
Sorry, I'm not sure what you mean with "all damage events".
How do I trigger this event?

**Comment by master117** [comment]
I mean player_hurt

**Comment by saul** [comment, marked as answer]
Am I right in thinking that you can only hold a single molotov at once? With that, you can assume that any damage given to victim V by attacker A using a molotov has come from that molotov in that round.

Unfortunately as the `player_hurt` event doesn't tell you the entity index of the weapon used to cause the damage, there's nothing more that can be done.
